pkg/util/tls_config.go

```diff
@@ -86,25 +88,25 @@ func GetMinTLSVersion() uint16 {
 	return minTLSVersion
 }
 
-func initMinTLSVersion(logger logr.Logger) uint16 {
-	version, found := os.LookupEnv("KEDA_HTTP_MIN_TLS_VERSION")
-	minVersion := tls.VersionTLS12
-	if found {
-		switch version {
-		case "TLS13":
-			minVersion = tls.VersionTLS13
-		case "TLS12":
-			minVersion = tls.VersionTLS12
-		case "TLS11":
-			minVersion = tls.VersionTLS11
-		case "TLS10":
-			minVersion = tls.VersionTLS10
-		default:
-			logger.Info(fmt.Sprintf("%s is not a valid value, using `TLS12`. Allowed values are: `TLS13`,`TLS12`,`TLS11`,`TLS10`", version))
-			minVersion = tls.VersionTLS12
-		}
+func initMinTLSVersion() (uint16, error) {
+	version, _ := os.LookupEnv("KEDA_HTTP_MIN_TLS_VERSION")
+
```

Comment:
We should keep the default version IMO because people using `kubectl apply` from releases will not have this env set